docs/sources/query/template_functions.md

```diff
@@ -594,7 +599,7 @@ Signature: `Now() time.Time`
 Example:
 
 ```template
-{{ now }}
+`{{ now }}`
```

Comment:
Do we want to change line 584 to be consistent with the your changes? It uses quotes for the template but escapes the quotes in the template, so should work, it's just inconsistent with the rest of the examples.

Reply:
Added :) Thank you